src/components/folders/ReorderingModal.jsx

```diff
@@ -17,15 +17,7 @@ import { deslugifyDirectory } from "utils"
 
 const ReorderingModal = ({ params, dirData, onProceed, onClose }) => {
   const { collectionName, subCollectionName } = params
-  const [dirOrder, setDirOrder] = useState([])
-
-  /** ******************************** */
-  /*     useEffects to load data     */
-  /** ******************************** */
-
-  useEffect(() => {
-    if (dirData) setDirOrder(dirData)
-  }, [dirData])
```

Comment:
Just checking - why this this useEffect removed? Does the ReorderingModal rerender without the useEffect if dirData changes?

Reply:
hmm this has subtly different behaviour actually but i didn't see a noticeable difference. i'll change it back just to be safe ._.